### PR TITLE
Create 02_how-do-i-secure-my-account-by-setting-up-2fa.md

### PR DESCRIPTION
**Pull request by sylee-**

### CODE REVIEW

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# How do I secure my account even more by setting up Two-Factor Authentication (2FA)?
```
Comment (nielsenramon) [inline 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md:1]: I would remove the `even more` part for now :D

Comment (nielsenramon) [reply 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md:1]: You are missing the number in front of the file name!

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Your profile name or email address plus a password is the first layer of protection for your profile.  2FA is an extra layer of security that protects your account from unauthorized access by requiring two different authentication types. FetLife offers this additional security to our members.
```
Comment (nielsenramon) [inline 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md:3]: Not in love with it, but we need to first get out the strong password article. It's way more important and already finished in the google doc.

`Having a [strong password](link to password article) is the first line of defence to keeping your account safe. Two-Factor Authentication (2FA) is an additional layer of security that protects your account even more. We highly recommend all our members to use 2FA to keep your account and its contents safe!`

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# How do I secure my account even more by setting up Two-Factor Authentication (2FA)?
+
+Your profile name or email address plus a password is the first layer of protection for your profile.  2FA is an extra layer of security that protects your account from unauthorized access by requiring two different authentication types. FetLife offers this additional security to our members. 
+
+Before you enable 2FA, please download and install an authentication application to your phone.  Examples of these applications are [Authy](https://authy.com/download/) or [Google Authenticator](https://duckduckgo.com/?q=google+authenticator+app).  There are other applications, but these two are the most common.
```
Comment (nielsenramon) [inline 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md:5]: This should be a first step.

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+**Important**: You risk locking yourself out of your profile if you do not download an authentication app before continuing with the next steps.
+
+You can turn on 2FA at any time by:
+- Visiting your settings page
```
Comment (nielsenramon) [inline 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md:10]: I would use numbered list here

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# How do I secure my account even more by setting up Two-Factor Authentication (2FA)?
+
+Your profile name or email address plus a password is the first layer of protection for your profile.  2FA is an extra layer of security that protects your account from unauthorized access by requiring two different authentication types. FetLife offers this additional security to our members. 
+
+Before you enable 2FA, please download and install an authentication application to your phone.  Examples of these applications are [Authy](https://authy.com/download/) or [Google Authenticator](https://duckduckgo.com/?q=google+authenticator+app).  There are other applications, but these two are the most common.
+
+**Important**: You risk locking yourself out of your profile if you do not download an authentication app before continuing with the next steps.
```
Comment (nielsenramon) [inline 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md:7]: Not true, without an authenticator app you can't setup 2fa so no risk here.

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md (hunk 1 above)
Comment (sylee-) [reply 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md]: Done

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md (hunk 2 above)
Comment (sylee-) [reply 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md]: Changed.

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md (hunk 4 above)
Comment (sylee-) [reply 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md]: Done

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md (hunk 3 above)
Comment (sylee-) [reply 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md]: Adjusted.

File: 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md (hunk 5 above)
Comment (sylee-) [reply 17_security/how-do-i-secure-my-account-even-more-by-setting-up-2fa.md]: Removed.

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# How can I secure my account with a strong password?
+
+The following guidelines are essential for having a safe browsing experience across all password protected sites you use.  We highly recommend doing all of the below:
```
Comment (nielsenramon) [inline 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md:3]: There a space too much between 1st and second sentence

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# How can I secure my account with a strong password?
```
Comment (nielsenramon) [inline 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md:1]: Filename will break the help section. make sure to add an underscore after the number!

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+The following guidelines are essential for having a safe browsing experience across all password protected sites you use.  We highly recommend doing all of the below:
+
+1. Choose unique passwords for each site that require a password
+2. Use a password manager to keep track of your passwords (for example: [1Password](https://1password.com/) or [LastPass](https://www.lastpass.com/). These password managers will also automatically tell you if a site has been compromised so that you can change your password on that site!
+3. Keep your passwords private and share them with no one, **not even your partner!**
+4. When creating a password, use a [password generator](https://1password.com/password-generator/) to help you
+5. When creating a password:
+-  Avoid common words and character combinations
```
Comment (nielsenramon) [inline 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md:10]: We will need to indent these list items with 2 spaces.

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+- The longer the password, the better! A minimum of 8 characters is a good start
+- Use a combination of letters (both capitalized and lowercase), numbers, and symbols
+6. If you didn’t choose a password manager, check whether your previous passwords have been compromised on other sites by using [Have I Been Pwnd](https://haveibeenpwned.com/Passwords)
+
+Even when you manage your passwords well, there’s still a small chance of them being compromised. So, think about people with access to your devices, for example. So we recommend securing your account even more by setting up [Two-Factor Authentication (2FA)](link to 2FA article).
```
Comment (nielsenramon) [inline 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md:15]: `Even when you manage your passwords well, there’s still a small chance of them being compromised. Think about people with access to your devices for example. So we recommend securing your account even more by setting up [Two-Factor Authentication (2FA)](link to 2FA article).`

File: 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+# How do I secure my account by setting up Two-Factor Authentication (2FA)?
+
+Having a [strong password](link to password article) is the first line of defence to keeping your account safe. Two-Factor Authentication (2FA) is an additional layer of security that protects your account even more. We highly recommend all our members to use 2FA to keep your account and its contents safe!
+
+In order to enable 2FA, please download and install an authentication application to your phone.  Examples of these applications are [Authy](https://authy.com/download/) or [Google Authenticator](https://duckduckgo.com/?q=google+authenticator+app).  There are other applications, but these two are the most common.
```
Comment (nielsenramon) [inline 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md:5]: Please remove duplicate spaces in these sentences!

Comment (nielsenramon) [reply 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md:5]: I'm not sure about linking to duckduckgo either btw. Not many people will be familiar with it. I would provide 2 links one for iOS and one for Android.

iOS: https://apps.apple.com/us/app/google-authenticator/id388497605

Let's remove this line as well: `There are other applications, but these two are the most common.`

File: 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+You can turn on 2FA at any time by:
+1. Visiting your settings page
+2. Selecting “Enable” next to 2FA
+3. Entering your password on the next screen
+4. Using your authentication app to scan the QR code
+5. Entering the verification code you receive from your authentication app to activate 2FA
+
+**Note**: After 2FA is enabled, you will be prompted to download a list of recovery codes.  Please keep those codes safe because they can help you log in if something happens to the device you used to enable 2FA.
+
+Alternatively, you can enable 2FA here https://fetlife.com/settings/account/2fa/enable/password
```
Comment (nielsenramon) [inline 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md:16]: I would move this above the note.

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md (hunk 7 above)
Comment (sylee-) [reply 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md]: @nielsenramon - I think I fixed this correctly.  I thought we weren't supposed to use any "capital" key strokes.  Would you please write out the proper filename if I did it wrong?

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md (hunk 6 above)
Comment (sylee-) [reply 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md]: Fixed

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md (hunk 8 above)
Comment (sylee-) [reply 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md]: Done

File: 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md (hunk 9 above)
Comment (sylee-) [reply 17_security/01-how-can-i-secure-my-account-with-a-strong-password.md]: Done.

File: 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md (hunk 10 above)
Comment (sylee-) [reply 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md:5]: I added links for Google Authenticator for iOS and Android.

File: 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md (hunk 11 above)
Comment (sylee-) [reply 17_security/02-how-do-i-secure-my-account-by-setting-up-2fa.md:16]: Moved.

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md [hunk 12]
```diff
@@ -0,0 +1,15 @@
+# How can I secure my account with a strong password?
+
+The following guidelines are essential for having a safe browsing experience across all password protected sites you use.  We highly recommend doing all of the below:
+
+1. Choose unique passwords for each site that require a password
+2. Use a password manager to keep track of your passwords (for example: [1Password](https://1password.com/) or [LastPass](https://www.lastpass.com/)). These password managers will also automatically tell you if a site has been compromised so that you can change your password on that site!
+3. Keep your passwords private and share them with no one, **not even your partner!**
+4. When creating a password, use a [password generator](https://1password.com/password-generator/) to help you
```
Comment (nielsenramon) [inline 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:8]: `When creating a password, use a [password generator](https://1password.com/password-generator/) to help you and:`

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+5. When creating a password:
```
Comment (nielsenramon) [inline 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:9]: Remove this line so point 4 has these bullet points below

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+  - Avoid common words and character combinations
+  - The longer the password, the better! A minimum of 8 characters is a good start
+  - Use a combination of letters (both capitalized and lowercase), numbers, and symbols
+6. If you didn’t choose a password manager, check whether your previous passwords have been compromised on other sites by using [Have I Been Pwnd](https://haveibeenpwned.com/Passwords)
```
Comment (nielsenramon) [inline 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:13]: Let's remove this line, too technical.

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+
+Even when you manage your passwords well, there’s still a small chance of them being compromised. So, think about people with access to your devices for example. So we recommend securing your account even more by setting up [Two-Factor Authentication (2FA)](link to 2FA article).
```
Comment (nielsenramon) [inline 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:15]: `Even when you manage your passwords well, there’s still a small chance of them being compromised (e.g. people with access to your devices). So we recommend securing your account even more by setting up [Two-Factor Authentication (2FA)](link to 2FA article).`

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md [hunk 16]
```diff
@@ -0,0 +1,15 @@
+# How can I secure my account with a strong password?
+
+The following guidelines are essential for having a safe browsing experience across all password protected sites you use.  We highly recommend doing all of the below:
```
Comment (nielsenramon) [inline 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:3]: Double space alert!

File: 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md [hunk 17]
```diff
@@ -0,0 +1,16 @@
+# How do I secure my account by setting up Two-Factor Authentication (2FA)?
+
+Having a [strong password](link to password article) is the first line of defence to keeping your account safe. Two-Factor Authentication (2FA) is an additional layer of security that protects your account even more. We highly recommend all our members to use 2FA to keep your account and its contents safe!
+
+In order to enable 2FA, please download and install an authentication application to your phone. Examples of these applications are [Authy](https://authy.com/download/), [Google Authenticator for iOS](https://apps.apple.com/us/app/google-authenticator/id388497605), or [Google Authenticator for Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US&gl=US).
```
Comment (nielsenramon) [inline 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md:5]: `... Examples of these applications are [Authy](https://authy.com/download/) or Google Authenticator ([iOS](https://apps.apple.com/us/app/google-authenticator/id388497605) or [Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US&gl=US)).`

File: 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+
+You can turn on 2FA at any time by:
+1. Visiting your settings page
```
Comment (nielsenramon) [inline 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md:8]: `Visiting your Account Settings`

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md (hunk 12 above)
Comment (sylee-) [reply 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:8]: Replaced

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md (hunk 13 above)
Comment (sylee-) [reply 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:9]: Done.

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md (hunk 15 above)
Comment (sylee-) [reply 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:15]: Done.

File: 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md (hunk 14 above)
Comment (sylee-) [reply 17_security/01_how-can-i-secure-my-account-with-a-strong-password.md:13]: Removed

File: 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md (hunk 18 above)
Comment (sylee-) [reply 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md]: Fixed.

File: 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md (hunk 17 above)
Comment (sylee-) [reply 17_security/02_how-do-i-secure-my-account-by-setting-up-2fa.md]: Got it.

**Review by ErebusBat — APPROVED** [review]
Looks good!

**Review by nielsenramon — APPROVED** [review]
@sylee- When merged, please keep an eye out when it's live so you can create a new PR and switch the links!

**Pull request merged**